site/troubleshooting.md

```diff
@@ -8,7 +8,7 @@ layout: default
 Make sure you are running the latest version - you can download it
 with
 
-    docker pull zettio/weave:latest
+    docker pull weaveworks/weave:latest
```

Comment:
we haven't changed the image names yet, so the above should remain unchanged too.